migrations/2018-11-23-113821_add_access_levels/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users ADD COLUMN access TEXT;
```

Comment:
The database column should be added with a default value and NOT NULL constraint:
```
ALTER TABLE users ADD COLUMN access TEXT DEFAULT 'User' NOT NULL;
```
This avoids lots of runtime checks from now on and ever after!

I would prefer type INTEGER instead of TEXT. A string is overkill for this purpose. If we don't expect more than a few different roles we could even encode this field as a bitmask, i.e. _Guest_ = 0, _User_ = 1, _Scout_ = 8, _Admin_ = 64 (with the option to add intermediate levels or a even a _Super Admin_ level later). This would allow us to assign multiple roles/entitlements/levels to a user in the future without introducing a separate relation. For the first 8 different values such a column occupies only a single byte of storage. Moreover with an integer column ordering in database queries is possible!